app/controllers/users.js

```diff
@@ -96,3 +96,13 @@ exports.renew = (req, res, next) => {
     next(errors.validationError('Invalid renewId'));
   }
 };
+
+exports.list = (req, res, next) => {
+  const limit = req.query.limit || 5;
+  const offset = req.query.offset * limit || 0;
```

Comment:
The param offset is actually how many items do you wanna skip before taking the desired limit. I guess you are mixing it with the idea of page that you should multiply it by the limit to get the actual offset.